acceptancetests/jujupy/k8s_provider/base.py

```diff
@@ -211,13 +211,14 @@ def patch_configmap(self, namespace, cm_name, key, value):
         cm['data'] = data
         self.kubectl_apply(json.dumps(cm))
 
-    def sh(self, *args):
+    def sh(self, *args, shell=False):
         args = [str(arg) for arg in args]
```

Comment:
If we are going to specify shell=True, we need to quote the args. See https://docs.python.org/3/library/subprocess.html#security-considerations

Reply:
ah, right. fixed